sass/_themes.scss

```diff
@@ -285,6 +296,7 @@ $themes: (
         CatDisplayCatContentTitleText: $COLOR_LIGHT_BLUE_1,
         CatDisplayCatContentIcon: $COLOR_LIGHT_BLUE_1,
         CatDisplayCatContentHover: $COLOR_LIGHT_BLUE_2,
+        CatDisplayCatContentHoverIcon: $COLOR_LIGHT_BLUE_3,
```

Comment:
@ashkanx @morteza-binary I add these new items for handling `Categorical display` design